example.rb

```diff
@@ -29,6 +30,6 @@
 end
 [thread1, thread2].each(&:join)
 span.finish
-LightStep.flush
+OpenTracing.global_tracer.flush
```

Comment:
maybe worth making a comment about `flush` being a LightStep extension (and not part of the OpenTracing standard per se)